src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -644,7 +645,7 @@ module Plonk_constraint = struct
         (eval_one : v -> f) (t : (v, f) t) =
       match t with
       (* cl * vl + cr * vr + co * vo + m * vl*vr + c = 0 *)
-      | Basic { l = cl, vl; r = cr, vr; o = co, vo; m; c } ->
+      | Basic { force = b; l = cl, vl; r = cr, vr; o = co, vo; m; c } ->
```

Comment:
Will eval still work in the case it's forced and it's a single generic gate?  Do you need to have behaviour here conditional on force?